onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -131,10 +131,27 @@
     ParseHtpPerformanceMode(htp_performance_mode_pos->second);
   }
 
-  qnn_backend_manager_ = std::make_unique<qnn::QnnBackendManager>(backend_path_,
-                                                                  profiling_level_,
-                                                                  rpc_control_latency_,
-                                                                  htp_performance_mode_);
+#ifndef NDEBUG
+  // Enable use of QNN Saver only in debug builds.
+  static const std::string QNN_SAVER_PATH_KEY = "qnn_saver_path";
```

Comment:
>qnn_saver_path [](start = 49, length = 14)

need to enable it for onnx_test_runner tool

Reply:
Done